components/gantt/gantt-tree/columns/menu.md

```diff
@@ -0,0 +1,401 @@
+        public string Title { get; set; }
+        public double PercentComplete { get; set; }
+        public DateTime Start { get; set; }
+        public DateTime End { get; set; }
+    }
+
+    public int LastId { get; set; } = 1;
+    List<FlatModel> Data { get; set; }
+
+    protected override void OnInitialized()
+    {
+        Data = new List<FlatModel>();
+        var random = new Random();
+
+        for (int i = 1; i < 6; i++)
+        {
+            var newItem = new FlatModel()
+                {
+                    Id = LastId,
+                    Title = "Employee  " + i.ToString(),
+                    Start = new DateTime(2020, 12, 10 + i),
+                    End = new DateTime(2020, 12, 11 + i),
+                    PercentComplete = Math.Round(random.NextDouble(), 2)
+                };
+
+            Data.Add(newItem);
+            var parentId = LastId;
+            LastId++;
+
+            for (int j = 0; j < 5; j++)
+            {
+                Data.Add(new FlatModel()
+                    {
+                        Id = LastId,
+                        ParentId = parentId,
+                        Title = "    Employee " + i + " : " + j.ToString(),
+                        Start = new DateTime(2020, 12, 20 + j),
+                        End = new DateTime(2020, 12, 21 + i + j),
+                        PercentComplete = Math.Round(random.NextDouble(), 2)
+                    });
+
+                LastId++;
+            }
+        }
+
+        base.OnInitialized();
+    }
+
+}
+````
+
+## Features
+
+To control the features of the Column Menu, use the `<GanttColumnMenuSettings>` tag, nested inside the `<GanttSettings>` tag.
+
+By default, all Column Menu features are enabled.
+
+### Column Chooser
+
+The Column Chooser in the Column Menu allows you to toggle the visibility of Gantt columns. By default, all columns are visible under the **Columns** section of the Column Menu. To expand the menu, click the **Columns** item.
+
+The **Apply** button sets the column visibility according to the current checkbox values and closes the column menu. The **Reset** button reverts the checkbox values to their state when the column menu was opened. At this point, the user can start over, click **Apply**, or click outside the column menu to close it.
+
+* To disable the column chooser, set the `ShowColumnChooser` parameter of the `<GanttColumnMenuSettings>` to `false`.
+* To hide a column from the Column Chooser, set the `VisibleInColumnChooser` property of the column to `false`.
+
+### Filtering
+
+To control whether filtering is possible from the Column Menu, set the `FilterMode` parameter of the `GanttColumnMenuSettings` tag to a member of the `ColumnMenuFilterMode` enum:
+
+* `None`—disables the filtering from the Column Menu. This is the recommended option if you use the [`FilterRow` mode]({%slug gantt-filter-row%}).
+* `FilterMenu`—enables filtering from a filter menu.
+
+### Groupable
```

Comment:
Better use "Grouping" to make it consistent with the other headings. There is no need to match the feature name, because "Filtering" and "Frozen columns" don't either.

Reply:
The Groupable paragraph as a whole is a mistake on my part, as the Gantt does not have such a feature.